README.md

```diff
@@ -286,7 +299,7 @@ Or you could include the index or an ID in your attribute:
 <li data-testid={`item-${item.id}`}>{item.text}</li>
 ```
 
-And then you could use the `queryByTestId`:
+And then you could use the `queryByTestId` utility:
```

Comment:
Could you switch this to `getByTestId`?

Also, could you add a FAQ for: `What if I want to verify that an element does NOT exist`? Where we show how to use `queryByTestId` as well as `container.querySelector` as possible solutions.